doc/source/development/building_from_source.rst

```diff
@@ -90,6 +84,35 @@ for the shared lib, *e.g.* ``set (GDAL_LIB_OUTPUT_NAME gdal_x64 CACHE STRING ""
 
 Refer to :ref:`using_gdal_in_cmake` for how to use GDAL in a CMake project.
 
+Building on Windows
++++++++++++++++++++
+
+On Windows, one may need to specify generator:
+
+.. code-block:: bash
+
+    cmake -G "Visual Studio 15 2017" ..
+
+
+Building on MacOS
++++++++++++++++++
+
+On MacOS, there are a couple of libraries that do not function properly when the GDAL build requirements are installed using Homebrew.
+
+The `Apache Arrow <https://arrow.apache.org/docs/index.html>`_ library included in the current distribution of CMake is broken, and causes a detection issue. In order to build GDAL successfuly, configure CMake to not find the Arrow package:
```

Comment:
Which Apache Arrow library is included in the current distribution of CMake? Was this meant to refer to Homebrew instead of CMake?

Reply:
> Was this meant to refer to Homebrew instead of CMake?

surely, fixed